tests/unit/src/test/scala/tests/AddPackageSlowSuite.scala

```diff
@@ -0,0 +1,88 @@
+package tests
+
+import java.nio.file.Files
+
+import scala.meta.internal.metals.RecursivelyDelete
+import scala.meta.internal.metals.MetalsEnrichments._
+
+object AddPackageSlowSuite extends BaseSlowSuite("add package") {
+
+  testAsync("single level") {
+    cleanCompileCache("a")
+    RecursivelyDelete(workspace.resolve("a"))
+    Files.createDirectories(
+      workspace.resolve("a/src/main/scala/a").toNIO
+    )
+    for {
+      _ <- server.initialize("""
```

Comment:
the formatting looks scewed - could you put the start of the string in the new line?

Reply:
I tried but scalafmt reformatted it again.

Reply:
Pretty sure that is not scalafmt, you can see in the SbtSlowSuite that it works.